RetroBar/PropertiesWindow.xaml.cs

```diff
@@ -277,6 +277,30 @@ private void AutoStartCheckBox_OnChecked(object sender, RoutedEventArgs e)
         private void cboLanguageSelect_SelectionChanged(object sender, System.Windows.Controls.SelectionChangedEventArgs e)
         {
             LoadVersion();
+
+            if (!EnvironmentHelper.IsWindows10OrBetter)
+            {
+                PopulateClockActionsForPreWindows10();
+            }
+        }
+
+        private void PopulateClockActionsForPreWindows10()
+        {
+            var availableClockActions = (FindResource("clock_click_action_values") as Array)?.Cast<object>().ToList();
+            if (availableClockActions == null)
+            {
+                return;
+            }
+
+            if (Settings.Instance.ClockClickAction > ClockClickOption.OpenAeroClockFlyout)
+            {
+                // ClockClickAction is out of range; reverting to default
+                cboClockAction.SelectedValue = availableClockActions[(int)ClockClickOption.OpenAeroClockFlyout];
```

Comment:
I wanted it to use the default function of the stock taskbar, but since this is inconsistent with the setting in previous releases, it has been reverted.